packages/utils/create-react-app/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@parcel/create-react-app",
+  "version": "2.0.0-beta.2",
+  "bin": {
+    "parcel-create-react-app": "src/bin.js"
```

Comment:
Should we try to get that npm package name?

Reply:
I think this is @devongovett 's decision since he'd probably have the most luck in getting it? But I can reach out if you think thats beneficial @mischnic